熔断组件，在项目不依赖openFeign时会启动报错

**Describe the bug**
`
Caused by: java.lang.ClassNotFoundException: org.springframework.cloud.openfeign.FeignAutoConfiguration
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[na:1.8.0_251]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[na:1.8.0_251]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355) ~[na:1.8.0_251]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[na:1.8.0_251]
	at java.lang.Class.forName0(Native Method) ~[na:1.8.0_251]
	at java.lang.Class.forName(Class.java:348) ~[na:1.8.0_251]
	at org.springframework.util.ClassUtils.forName(ClassUtils.java:284) ~[spring-core-5.2.13.RELEASE.jar:5.2.13.RELEASE]
	at org.springframework.util.ClassUtils.resolveClassName(ClassUtils.java:324) ~[spring-core-5.2.13.RELEASE.jar:5.2.13.RELEASE]
	... 25 common frames omitted

@ConditionalOnProperty(value = "spring.cloud.polaris.circuitbreaker.enabled",
		havingValue = "true", matchIfMissing = true)
@Configuration(proxyBeanMethods = false)
@AutoConfigureAfter(PolarisContextAutoConfiguration.class)
@AutoConfigureBefore(FeignAutoConfiguration.class)
public class PolarisFeignClientAutoConfiguration {

	@Bean
	public ConsumerAPI consumerAPI(SDKContext context) {
		return DiscoveryAPIFactory.createConsumerAPIByContext(context);
	}

	@Bean
	@Order(HIGHEST_PRECEDENCE)
	public PolarisFeignBeanPostProcessor polarisFeignBeanPostProcessor(
			ConsumerAPI consumerAPI) {
		return new PolarisFeignBeanPostProcessor(consumerAPI);
	}

}
`

**To Reproduce**
只依赖如下组件
`      <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
        </dependency>

        <dependency>
            <groupId>com.tencent.cloud</groupId>
            <artifactId>spring-cloud-starter-tencent-polaris-discovery</artifactId>
        </dependency>

        <dependency>
            <groupId>com.tencent.cloud</groupId>
            <artifactId>spring-cloud-starter-tencent-polaris-circuitbreaker</artifactId>
        </dependency>`

**Expected behavior**
可以正常启动，

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,3 +9,4 @@
 - [Upgrade: fix third-party lib CVEs & upgrade core spring libs version](https://github.com/Tencent/spring-cloud-tencent/pull/237)
 - [change escape way into encode](https://github.com/Tencent/spring-cloud-tencent/pull/251)
 - [feat:support reading configuration from application.yml or application.properties.](https://github.com/Tencent/spring-cloud-tencent/pull/259)
+- [fix:fix ClassNotFoundException while not importing openfeign when using circuit-breaker module.](https://github.com/Tencent/spring-cloud-tencent/pull/269)

--- a/spring-cloud-starter-tencent-polaris-circuitbreaker/src/main/java/com/tencent/cloud/polaris/circuitbreaker/PolarisFeignClientAutoConfiguration.java
+++ b/spring-cloud-starter-tencent-polaris-circuitbreaker/src/main/java/com/tencent/cloud/polaris/circuitbreaker/PolarisFeignClientAutoConfiguration.java
@@ -25,6 +25,7 @@ import com.tencent.polaris.factory.api.DiscoveryAPIFactory;
 
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
 import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.cloud.openfeign.FeignAutoConfiguration;
 import org.springframework.context.annotation.Bean;
@@ -42,6 +43,7 @@ import static org.springframework.core.Ordered.HIGHEST_PRECEDENCE;
 @ConditionalOnProperty(value = "spring.cloud.polaris.circuitbreaker.enabled",
 		havingValue = "true", matchIfMissing = true)
 @Configuration(proxyBeanMethods = false)
+@ConditionalOnClass(name = "org.springframework.cloud.openfeign.FeignAutoConfiguration")
 @AutoConfigureAfter(PolarisContextAutoConfiguration.class)
 @AutoConfigureBefore(FeignAutoConfiguration.class)
 public class PolarisFeignClientAutoConfiguration {

